spec/rails_helper.rb

```diff
@@ -56,6 +68,8 @@ def get(path, headers: nil, sign_with: nil, **args)
 end
 
 RSpec.configure do |config|
+  # This is set before running spec:system, see lib/tasks/tests.rake
+  config.filter_run_excluding type: :system unless RUN_SYSTEM_SPECS
```

Comment:
This is a good approach for this PR, which introduces this type.

If/when we do more migration/shuffling of the system/feature specs, we could migrate this to a specific tag/dir instead of all the system specs.